verilog/analysis/checkers/no_trailing_spaces_rule_test.cc

```diff
@@ -42,6 +42,7 @@ TEST(NoTrailingSpacesRuleTest, AcceptsText) {
       {"  xxx\n\n"},
       {"\txxx\n\n"},
       {"  xxx\n  yyy\n"},
+      {"  xxx\n  yyy\r\n"},
```

Comment:
Now that the end of line is two characters long, another good corner case to consider would be to check that two consecutive `\r\n` don't create trouble as an additional test-case.
`{"  xxx\n  yyy\r\n\r\n"},`

Reply:
Thank you, I've added the test case.